src/components/AddBook.js

```diff
@@ -20,12 +20,14 @@ const AddBook = () => {
   };
 
   return (
-    <form className="newbook" onSubmit={actionSubmit}>
-      <h2>ADD NEW BOOK</h2>
-      <input className="title" type="text" placeholder="Book title" value={title} onChange={(e) => setTitle(e.target.value)} />
-      <input className="author" type="text" placeholder="Book author" value={author} onChange={(e) => setAuthor(e.target.value)} />
-      <button type="submit">Add Book</button>
-    </form>
+    <div className="adding">
+      <h2 className="new-title">ADD NEW BOOK</h2>
+      <form className="newbook" onSubmit={actionSubmit}>
+        <input className="title" type="text" placeholder="Book title" value={title} onChange={(e) => setTitle(e.target.value)} />
+        <input className="author" type="text" placeholder="Book author" value={author} onChange={(e) => setAuthor(e.target.value)} />
```

Comment:
- [ ] Input values are not showing when ı write something in it. Please fix it.
- [ ] Kindly give space end of the form.

### ORIGINAL DESIGN
![image](https://user-images.githubusercontent.com/38635158/167258896-8ff57f48-615c-4da3-99c9-4cd950825f0a.png)